### PR TITLE
feat: add sounds

### LINKED ISSUE #1919: [client] add sounds
- hex selection in building mode/ travel mode
- when creating/accepting orders/ swapping
- when clicking on a building

### LINKED ISSUE #1924: [client] add screaming donkey sound when burning donkeys

### PR DESCRIPTION
**Pull request by edisontim**
Closes #1919 
Closes #1924

### CODE REVIEW

File: client/src/three/components/BuildingPreview.ts [hunk 1]
```diff
@@ -86,6 +88,11 @@ export class BuildingPreview {
 
   public setBuildingPosition(position: THREE.Vector3) {
     if (this.previewBuilding) {
+      if (!this.currentHexHovered || !this.currentHexHovered.equals(position)) {
+        new Audio(dir + soundSelector.hoverClick).play();
```
Comment (r0man1337) [inline client/src/three/components/BuildingPreview.ts:92]: There was `useShovelSound()` hook in useUiSounds file, that plays two similar sounds one by one.
This sound was designed exactly for hex selection in building mode.

May be we can reuse it here?

Comment (aymericdelab) [reply client/src/three/components/BuildingPreview.ts:92]: yeah let's do it

File: client/src/three/scenes/Hexception.tsx [hunk 2]
```diff
@@ -629,3 +650,29 @@ export default class HexceptionScene extends HexagonScene {
     });
   }
 }
+
+export const playBuildingSound = (buildingType: BuildingType | undefined) => {
```
Comment (aymericdelab) [inline client/src/three/scenes/Hexception.tsx:654]: shouldn't all this logic be in the useUISound file ?

File: client/src/three/components/BuildingPreview.ts (hunk 1 above)
Comment (edisontim) [reply client/src/three/components/BuildingPreview.ts:92]: Should I change it for the hex selection in the worldmap view as well?

**Pull request merged**